Frontend/Areas/MicrosoftIdentity/Pages/Account/AccessDenied.cshtml

```diff
@@ -1,10 +1,10 @@
 ﻿@page "/access-denied"
+@inject Microsoft.Extensions.Configuration.IConfiguration _configuration
 @model Microsoft.Identity.Web.UI.Areas.MicrosoftIdentity.Pages.Account.AccessDeniedModel
 @{
     Layout = "_Layout";
     ViewData["Title"] = "Access denied";
-    const string serviceDeskRequestLink = "https://dfe.service-now.com/serviceportal?id=sc_cat_item&sys_id=59d68b331bd13050199d6397b04bcb23";
 }
 
 <h1 class="govuk-heading-l">You do not have access to this service</h1>
-<p class="govuk-body">If you think you need access, submit a <a href="@serviceDeskRequestLink" class="govuk-link">Service Portal request</a>.</p>
\ No newline at end of file
+<p class="govuk-body">If you think you need access, <a href="@_configuration["AccessRequest"]" class="govuk-link">request access to this service</a>.</p>
```

Comment:
Perhaps the working could be changed - saying "If you think you need access, LINK request access to this service." Maybe even adding "...use this form to request access to this service"

Reply:
This is the content as requested by the design team :)